qa/build.gradle

```diff
@@ -5,6 +5,9 @@ subprojects { Project subproj ->
   subproj.tasks.withType(RestIntegTestTask) {
     subproj.extensions.configure("${it.name}Cluster") { cluster ->
       cluster.distribution = System.getProperty('tests.distribution', 'oss-zip')
+      if (cluster.distribution == 'integ-test-zip') {
```

Comment:
I think we're better off checking if the system property is `integ-test-zip` in the cluster formation tasks and failing there. Basically, the only way you can get `integ-test-zip` is to not set the property.